src/Item.java

```diff
@@ -15,16 +18,23 @@ public String getItemName() {
 	public void setItemName(String itemName) {
 		this.itemName = itemName;
 	}
-	public double getItemPrepTime() {
-		return itemPrepTime;
+	
```

Comment:
Override Equals and Hashcode

```suggestion
		@Override
	public int hashCode() {
		final int prime = 31;
		int result = 1;
		result = prime * result + itemId;
		result = prime * result + ((itemName == null) ? 0 : itemName.hashCode());
		long temp;
		temp = Double.doubleToLongBits(itemRate);
		result = prime * result + (int) (temp ^ (temp >>> 32));
		return result;
	}
	@Override
	public boolean equals(Object obj) {
		if (this == obj)
			return true;
		if (obj == null)
			return false;
		if (getClass() != obj.getClass())
			return false;
		Item other = (Item) obj;
		if (itemId != other.itemId)
			return false;
		if (itemName == null) {
			if (other.itemName != null)
				return false;
		} else if (!itemName.equals(other.itemName))
			return false;
		if (Double.doubleToLongBits(itemRate) != Double.doubleToLongBits(other.itemRate))
			return false;
		return true;
	}
```